T/Torch/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Torch"
+version = v"0.1.0"
```

Comment:
What does this version number refer to?  We prefer to use the same version number as the upstream package, which in this case should be Torch I assume?

Reply:
The package is at github.com/dhairyagandhi96/Torch.jl which hasn't been registered yet pending getting the jll so the first user experience is decent. Happy to change it appropriately

Reply:
i think I'm confused.  The name is "Torch", doesn't it refer to [this Torch](https://pytorch.org/)?

Reply:
Right, so the Julia package is a wrapper around that torch's kernels

Reply:
Then the version number should refer to the Torch version used as source.  We don't care about the Julia package that will wrap this, these binaries can be used also outside of the Julia ecosystem.

Reply:
Fair enough, will change